content/posts/en/eth2-staking-with-k8s-prysm.md

```diff
@@ -355,43 +366,44 @@ On the machine you plan to run NFS:
 ```
 
 ### Prepare Validator Wallets
 
-Please refer to the Prysm’s [official documentation](https://docs.prylabs.network/docs/mainnet/joining-eth2/#step-4-import-your-validator-accounts-into-prysm).
+Please refer to Prysm’s [official documentation](https://docs.prylabs.network/docs/mainnet/joining-eth2/#step-4-import-your-validator-accounts-into-prysm).
 
-Let’s get back to the NFS server. We need to configure the wallet directories that we created in the previous section. Before proceeding, please have your validator keys placed on your NFS machine. We use `$HOME/eth2.0-deposit-cli/validator_keys` as the example path to the validator keys. To create a Prysm’s wallet and import your validator keys into it, we use the Prysm startup script.
+Let’s get back to the NFS server. We need to configure the wallet directories that we created in the previous section. Before proceeding, please have your validator keys placed on your NFS machine. We use `$HOME/eth2.0-deposit-cli/validator_keys` as the example path to the validator keys. To create a wallet and import your validator keys for Prysm validator client, we use Prysm’s startup script.
 
 1. Please follow Prysm’s [documentation](https://docs.prylabs.network/docs/install/install-with-script/#downloading-the-prysm-startup-script) to download Prysm startup script.
 
-2. Execute the startup script. We have put the validator keys under `$HOME/eth2.0-deposit-cli/validator_keys`which is the key directory we are using in the following command.
+2. Execute the startup script. We placed the validator keys under `$HOME/eth2.0-deposit-cli/validator_keys`which is the key directory we are using in the following command.
 
     ```bash
     sudo ./prysm.sh validator accounts import --keys-dir=$HOME/eth2.0-deposit-cli/validator_keys
     ```
 
-3. When prompt, enter your wallet directory, for example `/data/prysm/wallet-1`
+3. When prompted, enter your wallet directory. For example, `/data/prysm/wallet-1`
 
-4. Create wallet password (back it up somewhere safe)
+4. Create the wallet password (**remember to back it up somewhere safe!**)
 
-5. Enter the password you used to create the validator keys with the [eth2.0-deposit-cli](https://github.com/ethereum/eth2.0-deposit-cli). If you enter it correctly the accounts will be imported into the new wallet.
+5. Enter the password you used to create the validator keys with the [eth2.0-deposit-cli](https://github.com/ethereum/eth2.0-deposit-cli). If you enter it correctly, the accounts will be imported into the new wallet.
 
 ### Change the owner of the data folder
 
-On the NFS machine, let’s change the directories ownership so later these directories can be mounted by Kubernetes as the storage volumes for the pods running the beacon node and the validator.
+On the NFS machine, let’s change the directories ownership so later these directories can be mounted by Kubernetes as the storage volumes for the pods running the beacon node and the validator. 
```

Comment:
the **directories'** ownership?

Reply:
as discussed, changed to **directory owner**

Reply:
It's not changed as well.